### PR TITLE
[CS] Add visual queue when training is done

### LINKED ISSUE #572: [CS] Add visual queue when training is done
Add visual queue when training is done

Current plan:
- EDJE have training, done status (and styles accordingly)
- Send signal done when training is done

### LINKED ISSUE #574: [CS] Add go to main menu
`go to main` behavior should be added training / inferencing pages.

The behavior should be equivalent to clear the naviframe and push main frame.

### PR DESCRIPTION
**Pull request by zhoonit**
**Changes proposed in this PR:**
- Visually notify when training is done
- Add go to main menu when training is done
- Go to main menu button shows the final best accuracy

Resolves #572, Resolves #574

**Self evaluation:**
1. Build test: [X]Passed [ ]Failed [ ]Skipped
2. Run test: [X]Passed [ ]Failed [ ]Skipped

Signed-off-by: Jihoon Lee <jhoon.it.lee@samsung.com>

### CODE REVIEW

**Review by jijoongmoon — APPROVED** [review]
LGTM

File: Applications/Tizen_native/CustomShortcut/src/main.c [hunk 1]
```diff
@@ -31,14 +31,32 @@ static int routes_to_(appdata_s *ad, const char *source) {
...
  * @param data
  */
-static void add_back_cb_(void *data) {
+static void notify_train_done(void *data) {
   appdata_s *ad = (appdata_s *)data;
+
+  char buf[256];
+
+  const char *source = "train_result";
+  sleep(1);
```
Comment (kparichay) [inline Applications/Tizen_native/CustomShortcut/src/main.c:44]: what is the purpose of this sleep?

Comment (zhoonit) [reply Applications/Tizen_native/CustomShortcut/src/main.c:44]: After training is done, before moving to training done user can see the last epoch result a bit longer to queue user that it has been finished. This enables smoother transition

Comment (kparichay) [reply Applications/Tizen_native/CustomShortcut/src/main.c:44]: Lets add a small comment here then.

**Review by kparichay — APPROVED** [review]
LGTM

File: Applications/Tizen_native/CustomShortcut/src/main.c (hunk 1 above)
Comment (zhoonit) [reply Applications/Tizen_native/CustomShortcut/src/main.c:45]: Done

**Pull request merged**